docs/03-extensions/adobe-b2b/index.mdx

```diff
@@ -139,3 +138,44 @@ documents will no longer be available if this feature is disabled
   [example override](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/theme-chocolatine/theme/modules/ProductView/Synthesis/AddProductToCart.jsx)
 
 :::
+
+## Known issues
+
+<!--------------------DETAILS-------------------->
+
+```mdx-code-block
+<details>
+  <summary className="flex justify-between">Bundle products can't be used in requisition lists</summary>
+  <div>
+```
+
```

Comment:
**issue:** I think a first "TLDR" paragraph is needed, so readers can have the main takeaway immediately

**suggestion:** 
```suggestion
Due to an issue (documented below) with Adobe B2B GraphQL layer, we don't recommend to use Requistion List with bundle items with the default Adobe B2B codebase. Please contact your Adobe support team if you plan to use this feature.

```